apps/meteor/server/models/raw/LivechatVisitors.ts

```diff
@@ -49,7 +49,11 @@ export class LivechatVisitorsRaw extends BaseRaw<ILivechatVisitor> implements IL
 		return this.findOne(query);
 	}
 
-	findOneGuestByEmailAddress(emailAddress: string): Promise<ILivechatVisitor | null> {
+	async findOneGuestByEmailAddress(emailAddress: string): Promise<ILivechatVisitor | null> {
+		if (!emailAddress) {
```

Comment:
trusting issues? 😛

Reply:
We'll fix this [here](https://rocketchat.atlassian.net/browse/SCI-176?atlOrigin=eyJpIjoiZWY5MTM0OGYzYjVmNGM1Y2FiNGU2ZmYwZDU5NzI4M2IiLCJwIjoiaiJ9)